content/blog/2016/2016-11-16-blueocean-beta-declarative-pipeline-pipeline-editor.adoc

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Announcing the Blue Ocean beta, Declarative Pipeline and Pipeline Editor"
+tags:
+- blueocean
+- ux
+- pipeline
+- jenkinsworld
+author: i386
+---
+
+At Jenkins World on Wednesday 14th of September the Jenkins project was happy to
+introduce the beta release of Blue Ocean. Blue Ocean is the new user experience
+for Jenkins is built from the ground up to take advantage of Jenkins Pipeline
+and has rethought entirely the way that modern developers will use Jenkins.
+
+Blue Ocean is available today via the Jenkins update center for Jenkins users
+running 2.7.1 and above.
+
+=== Get the beta
+
+Just search for _BlueOcean beta_ in the plugin manager, install it,
+browse to the dashboard then click the _Try BlueOcean UI_ button on the dashboard.
+
+=== Whats included?
+
+link:/blog/2016/05/26/introducing-blue-ocean/[Back in April we open sourced Blue Ocean]
+and shared our vision with the community. We're very happy that all the things we showed you then have
+shipped in the beta (software projects run *on time*?!).
+
+For a refresher on Blue Ocean, watch this short video:
+++++
+<center>
+<iframe width="853" height="480"
+src="https://www.youtube-nocookie.com/embed/3dITffteCD4?rel=0" frameborder="0"
+allowfullscreen></iframe>
+</center>
+++++
+
+=== Declarative Pipeline
+
+We heard a lot from the community about the usability of link:/doc/pipeline[Jenkins
+Pipeline]. Much of the feedback we received was boiled down to a desire to
```

Comment:
s/was// - if it's "was boiled down", then I want to know what did the boiling down. =) (yes, I'm nit-picky about verb tenses)